src/lambda/lambdaCheckParameters.ts

```diff
@@ -0,0 +1,13 @@
+export const lambdaCheckParameters = (object: object, indexes: string[]): object => {
+  const errors = {}
```

Comment:
What I recommend to make to improve this function:

- Do an early return, make the if already returning `{ object: undefined }`
- And later, you can just return a index.reduce returning the errors or the empty object

@Adapcon/bomb-squad @Adapcon/coca-squad  what you guys think about that?

Reply:
Done!!! :D